rs/nns/governance/src/governance.rs

```diff
@@ -4110,7 +4127,7 @@ impl Governance {
         // to have Open status while it does not accept votes anymore, since
         // the status change happens below this point.
         if proposal.status() == ProposalStatus::Open
-            || proposal.accepts_vote(now_seconds, voting_period_seconds)
+            || !proposal.voting_is_finished(now_seconds, voting_period_seconds)
```

Comment:
This is a good point.  I think we want to always recompute the tally, bc the only way for votes to be initiated is if the initial voter makes the request while accepts_vote() is true.  So we don't actually need this condition.  Any time a vote is registered and would trigger following, it would be completely valid to recompute the tally.  

A better optimization would be to keep the tally up to date as votes are made, but we'd need to do further analysis.

Reply:
We fixed this by moving (in another PR) the recalculating of the tally into cast_votes_and_cascade_follow since that's the only time the tally could actually be affected.